Automattic/wp-calypso
# Devdocs: Update notice component documentation to display example for is-transparent-info status

Related to https://github.com/Automattic/wp-calypso/issues/83533

## Proposed Changes

* Update Calypso Devdocs `Notice` component to display example for newly added `is-transparent-notice` status

![CleanShot 2023-11-06 at 16 48 53@2x](https://github.com/Automattic/wp-calypso/assets/10482592/4580f800-9c80-46d9-bc0a-cf7374e64621)


## Testing Instructions



* Checkout branch (or use live link)
* Navigate to `/devdocs/design/notice`
* Scroll down to the bottom of the Notice examples, and verify the `is-transparent-info` Notice example is displaying properly.

## Pre-merge Checklist
